ms-engine/src/main/java/net/trajano/ms/engine/SpringJaxRsHandler.java

```diff
@@ -240,10 +244,13 @@ public SpringJaxRsHandler(final ConfigurableApplicationContext baseApplicationCo
                 applicationContext.register(clazz);
             });
             reflections.getTypesAnnotatedWith(Provider.class).forEach(clazz -> applicationContext.register(clazz));
+            reflections.getTypesAnnotatedWith(RequestScoped.class).forEach(clazz -> applicationContext.register(clazz));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)